Usiqee/Module/Account/AccountDetails/AccountDetailsView.swift

```diff
@@ -92,3 +136,65 @@ extension AccountDetailsView {
         delegate?.didTapSettings()
     }
 }
+
+// MARK: - UITableViewDataSource
+extension AccountDetailsView: UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        if segmentedMenu.selectedItem == 0 {
+            let numberOfItems = musicalEntities.count
+            if numberOfItems == 0 {
+                tableView.setEmptyMessage(L10N.UserDetails.followedEmptyListMessage)
+            }else {
+                tableView.restore()
+            }
+            return numberOfItems
+        }
+        
+        tableView.restore()
+        return 0
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        if segmentedMenu.selectedItem == 0 {
```

Comment:
prefer guard let here

Reply:
its temporary, it will be changed to switch when news will be added

Reply:
OKay 👍